// Copyright 2019 The GraphicsFuzz Project Authors
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     https://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto3";

package gfauto;


message Binary {
  string name = 1;
  string platform = 2;
  string path = 3;
}

message Archive {
  string url = 1;
  string output_file = 2;
  string output_directory = 3;
}

message ArchiveSet {
  repeated Archive archives = 1;
  repeated Binary binaries = 2;
}
